app/models/user.rb

```diff
@@ -33,4 +33,11 @@ def is_member_or_admin?
     admin? || is_member?
   end
 
+  def is_company_owner?
+    if has_company?
+      company = Company.find(self.company.id)
+      company.company_number == membership_application.company_number
```

Comment:
This will do the same as `has_company`

If a membership_application has an associated company (`membership_application.company_id` refers to a `company.id`)

then `has_company` will be true and your method will be true.

Any and all` membership_application`s that refer to a company will return true. 

We don't have any concept of a specific _owner_ for a company.  Should we?